tests/addnode.py

```diff
@@ -12,6 +12,43 @@
 from loguru import logger as LOG
 
 
+def remove_node(primary, node_id):
```

Comment:
We call this `retire` everywhere else, it should probably be `retire_node` here as well.